test/torchaudio_unittest/functional/functional_impl.py

```diff
@@ -582,6 +583,21 @@ def test_rnnt_loss_costs_and_gradients_random_data_with_numpy_fp32(self):
             ref_costs, ref_gradients = rnnt_utils.compute_with_numpy_transducer(data=data)
             self._test_costs_and_gradients(data=data, ref_costs=ref_costs, ref_gradients=ref_gradients)
 
+    def test_apply_beamforming(self):
```

Comment:
Tests in `functional_impl.py` implements test logics specific to each tested modules, so please add docstring of what is the expectation of this test.

test docstring should tell future maintainers (without context) what it is testing, so often it is the form of "(under this condition), given this input, this out comes should happen".